diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9ac7020..4d148df 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -497,6 +497,7 @@ VP_OPTION(INSTALL_ANDROID_EXAMPLES "" "" "Install Android examples"  ""
 # Build python bindings as an option
 VP_OPTION(BUILD_PYTHON_BINDINGS  "" "" "Build Python bindings" "" ON IF (PYTHON3INTERP_FOUND AND USE_PYBIND11 AND NOT CMAKE_NOT_OK_FOR_BINDINGS AND VISP_PYTHON_INTERPRETER_ALLOWED AND NOT PYTHON3_NOT_OK_FOR_BINDINGS AND NOT CXX_STANDARD_NOT_OK_FOR_BINDINGS) )
 VP_OPTION(BUILD_PYTHON_BINDINGS_DOC  "" "" "Build the documentation for the Python bindings" "" ON IF BUILD_PYTHON_BINDINGS )
+VP_OPTION(GENERATE_PYTHON_STUBS  "" "" "Generate Python stubs" "" ON IF BUILD_PYTHON_BINDINGS )
 
 
 # Build demos as an option.
diff --git a/modules/python/CMakeLists.txt b/modules/python/CMakeLists.txt
index fc92632..924e7e9 100644
--- a/modules/python/CMakeLists.txt
+++ b/modules/python/CMakeLists.txt
@@ -109,13 +109,21 @@ set(VISP_PYTHON_BINDINGS_BUILD_PATH "${CMAKE_CURRENT_BINARY_DIR}/bindings")
 add_subdirectory(bindings)
 
 # Step 4: Copy stubs dir and install stubs for autocompletion
-add_subdirectory(stubs)
+if(GENERATE_PYTHON_STUBS)
+  add_subdirectory(stubs)
+  # Global target: compile and install the Python bindings
+  add_custom_target(
+    visp_python_bindings
+    DEPENDS visp_python_bindings_stubs
+  )
+else()
+  add_custom_target(
+    visp_python_bindings
+    DEPENDS visp_python_bindings_install
+  )
+endif()
+
 
-# Global target: compile and install the Python bindings
-add_custom_target(
-  visp_python_bindings
-  DEPENDS visp_python_bindings_stubs
-)
 
 # Step 5: Build documentation
 if(BUILD_PYTHON_BINDINGS_DOC)
