From 404e9ec40b9ba01416512182b9bf953bc0fcf67b Mon Sep 17 00:00:00 2001
From: Olivier Roussel <olivier.roussel@inria.fr>
Date: Tue, 5 Sep 2023 11:08:14 +0200
Subject: [PATCH] Set VISP_LINKER_LIBS as private.

---
 cmake/VISPModule.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/VISPModule.cmake b/cmake/VISPModule.cmake
index 855d6225e..97d89a969 100644
--- a/cmake/VISPModule.cmake
+++ b/cmake/VISPModule.cmake
@@ -808,8 +808,8 @@ macro(_vp_create_module)
       ${VISP_MODULE_${the_module}_DEPS_TO_LINK}
       ${VISP_MODULE_${the_module}_DEPS_EXT}
       ${VISP_MODULE_${the_module}_LINK_DEPS}
-      ${VISP_LINKER_LIBS}
     LINK_PRIVATE
+      ${VISP_LINKER_LIBS}
       ${VISP_MODULE_${the_module}_PRIVATE_REQ_DEPS}
       ${VISP_MODULE_${the_module}_PRIVATE_OPT_DEPS})
   add_dependencies(visp_modules ${the_module})
-- 
2.42.0

