From 2b5ce3b727647da73fefc1dbc02f8b81d91b796e Mon Sep 17 00:00:00 2001
From: Olivier Roussel <olivier.roussel@inria.fr>
Date: Mon, 9 Feb 2026 17:09:34 +0100
Subject: [PATCH] clang-cl on win does not provide unistd header

---
 3rdparty/clapack/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/3rdparty/clapack/CMakeLists.txt b/3rdparty/clapack/CMakeLists.txt
index d78c2819e..7ffc15faf 100644
--- a/3rdparty/clapack/CMakeLists.txt
+++ b/3rdparty/clapack/CMakeLists.txt
@@ -11,7 +11,7 @@ set(LAPACK_MINOR_VERSION 2 PARENT_SCOPE)
 set(LAPACK_PATCH_VERSION 1 PARENT_SCOPE)
 
 add_definitions(-DNO_BLAS_WRAP)
-if(WIN32 AND MSVC)
+if(WIN32 AND (MSVC OR CMAKE_CXX_COMPILER_ID MATCHES "Clang"))
   add_definitions(-DMSDOS -DNO_ISATTY)
 endif()
 
-- 
2.43.0

